mitodl/bootcamp-ecommerce
# Added klass payment statement

#### What are the relevant tickets?
Closes #8 

#### What's this PR do?
Adds the printable payment statement page

#### How should this be manually tested?
Navigate to your `/pay` page with some valid payments (`Order` with status `fulfilled` and some associated `Line`s), click "View Statement", review the page, confirm accuracy, observe how print-friendly it is

#### Where should the reviewer start?
Probably `bootcamp/templates/bootcamp/statement.html`

#### Screenshots (if appropriate)
<img width="728" alt="ss 2017-05-12 at 13 21 14" src="https://cloud.githubusercontent.com/assets/14932219/26009255/f89350d4-3715-11e7-9362-489aebbfad74.png">

PDF-printed version: [MIT Bootcamps - Payment Statement.pdf](https://github.com/mitodl/bootcamp-ecommerce/files/997461/MIT.Bootcamps.-.Payment.Statement.pdf)

#### What GIF best describes this PR or how it makes you feel?
<img src="http://s3.amazonaws.com/theoatmeal-img/comics/printers/header.png" width="300" />
